packages/notification/src/component/FormNotification.js

```diff
@@ -49,7 +49,7 @@ export const FormNotification = ({ setPageName }) => {
           _text={{ color: "black", textTransform: "capitalize" }}
           px="5"
           py="1"
-          bg="viewNotification.600"
+          bg="viewNotification.500"
```

Comment:
As discussed earlier - we should move towards a theme approach.

Reply:
we cant use the `.color` there is controled by `50,100,200,..,900` that's why use `.500`